src/royalty/ABClaim.sol

```diff
@@ -291,67 +291,76 @@ contract ABClaim is Initializable, AccessControlUpgradeable {
         // abKycModule.beforeRoyaltyClaim(_user, _signature);
     }
 
-    function _claimMultiDrop(
+function _claimMultiDrop(
         uint256[] calldata _dropIds,
         uint256[][] calldata _tokenIds,
         address _user,
         bytes calldata _signature
     ) internal {
         // Enforce KYC Signature validity through ABKYCModule contract
         _beforeClaim(_user, _signature);
-
-        // Check parameters validity
-        uint256 dLength = _dropIds.length;
-        if (dLength != _tokenIds.length) revert ABErrors.INVALID_PARAMETER();
+         // Check parameters validity
+        if (_dropIds.length != _tokenIds.length) revert ABErrors.INVALID_PARAMETER();
 
         uint256 totalClaimable;
 
-        for (uint256 i; i < dLength;) {
-            uint256 dropId = _dropIds[i];
-
+        for (uint256 i; i < _dropIds.length;) {
```

Comment:
have you tried to store _dropIds.length in a var ? 
does it give you stack too deep ?

if not, it could gas optimize a little